config/site.js

```diff
@@ -1,24 +1,71 @@
 exports.configSchema = {
-  basicAuth:
-    [
-      {
-        key: 'active',
-        type: 'boolean',
-        default: false,
-        label: "Is active?"
-      },
-      {
-        key: 'user',
-        type: 'string',
-        default: 'openstad',
-        label: "Username"
-      },
-      {
-        key: 'password',
-        type: 'string',
-        default: process.env.BASIC_AUTH_DEFAULT_PW,
-        label: "Password"
-      },
+  project: [
+    {
+      key: 'endDate',
+      type: 'date',
+      default: null,
+      label: "Project end date"
+    },
+  ],
+
+  anonymize: [
+    {
+      key: 'anonimizeUsersXDaysAfterEndDate',
```

Comment:
I see `anonymize` and `anonimize` used interchangeably throughout the code, can we change all these occurrences with `anonymize` (https://www.merriam-webster.com/dictionary/anonymize) or will this lead to breaking changes? (Might still be worth to have everything use the same wording)

Reply:
Yes, they should of course use the same wording. Fixed, tnx.